dataactvalidator/scripts/loadSf133.py

```diff
@@ -0,0 +1,193 @@
+import os
+import boto
+import glob
+import logging
+import re
+from collections import namedtuple
+
+import pandas as pd
+
+from dataactvalidator.app import createApp
+from dataactvalidator.scripts.loaderUtils import LoaderUtils
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models.domainModels import SF133
+from dataactcore.config import CONFIG_BROKER
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
Just a note here that, as we slowly build out better logging, we likely want to move this configuration line. As written, this'll set the (global) logger configuration whenever the module is first imported. We'll want to instead configure the logging closer to the entry points of the application (e.g. in the `app.py` files, and in the `if __name__ == '__main__':` portions of shell scripts). In many ways, logging is similar to having that global database connection -- you don't want to re-create it again and again if you can configure it once and keep referencing that.